Library/Homebrew/utils/ruby.sh

```diff
@@ -0,0 +1,46 @@
+setup-ruby-path() {
+  local HOMEBREW_VENDOR_RUBY_OPT;
+  local HOMEBREW_VENDOR_RUBY_PATH;
```

Comment:
The semicolons are unnecessary and we mostly try to use lowercase names for local variables to distinguish them from global variables (all caps) and exported variables (all caps and with a `HOMEBREW_` prefix).